ui-tests/src/test/java/org/openmrs/reference/page/EditVisitNotePage.java

```diff
@@ -10,15 +10,28 @@
 package org.openmrs.reference.page;
 
 import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
 
 public class EditVisitNotePage extends VisitNotePage {
-
+	
+    private static final By NOTE = By.id("w10");
+    private static final String  note = "this is edited note";
```

Comment:
cc @kdaud  did you see this commit

Reply:
The resource I shared [above](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/450#discussion_r681447476), my point was on `note` vs `NOTE`

Reply:
Adding some thing minor: I would start my string value with Uppercase letter, something like "`This is edited note`."

Reply:
> my point was on `note` vs `NOTE`

Please there is a difference between note and Note, i used note to differentiate it from NOTE which is line 17.

Reply:
> i used note to differentiate it from NOTE which is line 17.

In java programming variable naming convention a `static final variable` should be in `UPPERCASE`. See the reference again [here](https://stackoverflow.com/questions/7259687/java-naming-convention-for-static-final-variables)

Reply:
Thats  a simply a string whether its upper case or lower case its fine, as long as you use it how you have called it , please check a number of other areas where such kind of work is being used. Using  **note** to have a clear readable difference with NOTE .

Reply:
@sherrif10 did you get a chance to look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions

Reply:
Yeah , Let me commit it

Reply:
`Class and instance variables should follow Java specs. Any constants should be written in uppercase letters.`

Reply:
Resolved thanks @dkayiwa

Reply:
Resolved Thanks